src/fprime/fbuild/settings.py

```diff
@@ -78,6 +78,7 @@ class IniSettings:
             SettingType.PATH,
             lambda settings: settings["settings_file"],
         ),
+        ("default_cmake_options", SettingType.STRING, ""),
```

Comment:
Format this line like the other entries.

Reply:
Cannot do so.  The python formatter has chosen this way.